api-docs/list-clients.md

```diff
@@ -25,10 +25,13 @@ Notes:
       "firstName": "Mario",
       "lastName": "Luigi",
       "zip": "84107",
-      "birthday": "1981/01/01",
+      "birthday": "1981-01-01",
       "primaryPhone": "5551111111",
-      "dateAdded": "2019/05/13",
-      "createdBy": "Shigeru Miyamoto"
+      "dateAdded": "2019-05-13",
+      "createdBy": {
+        "userId": 123,
```

Comment:
Always nice to have the userId as well so that we can link to the user or fetch more info about them if needed.